docs/design/run.md

```diff
@@ -0,0 +1,268 @@
+# Introducing JJ run
+
+Authors: [Philip Metzger](mailto:philipmetzger@bluewin.ch), [Martin von Zweigberk](mailto:martinvonz@google.com), [Danny Hooper](mailto:hooper@google.com), [Waleed Khan](mailto:me@waleedkhan.name)
+
+Initial Version, 10.12.2022 (view full history [here](https://docs.google.com/document/d/14BiAoEEy_e-BRPHYpXRFjvHMfgYVKh-pKWzzTDi-v-g/edit))
+
+
+**Summary:** This Document documents the design of a new `run` command for 
+Jujutsu which will run a user-provided command or script across multiple revisions. 
+This can be used for running build commands, linters or formatters. For more 
+details, read the [Use-Cases of jj run.](#Use-Cases-of-jj-run).
```

Comment:
There's an extraneous period before the `]`.

Reply:
removed